utils/cPoint.h

```diff
@@ -16,8 +16,16 @@ class cPoint {
 
     bool isWithinRectangle(const cRectangle *pRectangle) const;
 
+    /**
+     * Helper class to split a point that is returned from a function into its two coordinates.
+     * It stores two references to the coordinate variables. The copy operator copies the coordinates
+     * from the given point into the variables pointed to by the references.
+     * Example:
+     *   int x, y;
+     *   cPoint::split(x, y) = getPoint();
```

Comment:
aha this makes sense. So this is a convenient way to get the two raw x,y coordinates without `.x` or `.y` 'ing this. Nice.